PowerUp/app/src/main/java/powerup/systers/com/kill_the_virus_game/KillTheVirusGame.java

```diff
@@ -83,14 +87,31 @@ public void clickScreen() {
      * Start the countdown timer
      */
     private void initialSetup() {
-        virus1Animator = animateVirus(imgVirus1, TimeUnit.SECONDS.toMillis(5), 0, 359);
-        virus2Animator = animateVirus(imgVirus2, TimeUnit.SECONDS.toMillis(5), 45, 404);
-        virus3Animator = animateVirus(imgVirus3, TimeUnit.SECONDS.toMillis(5), 90, 449);
-        virus4Animator = animateVirus(imgVirus4, TimeUnit.SECONDS.toMillis(5), 135, 494);
-        virus5Animator = animateVirus(imgVirus5, TimeUnit.SECONDS.toMillis(5), 180, 539);
-        virus6Animator = animateVirus(imgVirus6, TimeUnit.SECONDS.toMillis(5), 225, 584);
-        virus7Animator = animateVirus(imgVirus7, TimeUnit.SECONDS.toMillis(5), 270, 629);
-        virus8Animator = animateVirus(imgVirus8, TimeUnit.SECONDS.toMillis(5), 315, 674);
+        boolean calledByTutorialActivity = getIntent().getBooleanExtra(PowerUpUtils.CALLED_BY, false);
+        if(!calledByTutorialActivity){
```

Comment:
The if codition checks if the game id launched for first time or not and sets the values of the variables accordingly.